test/src/test/java/org/corfudb/integration/LogReplicationListenerIT.java

```diff
@@ -475,10 +458,6 @@ public void testMultipleClients() throws Exception {
         Assert.assertEquals(numUpdates, lrListener.getExistingEntries().size());
         Assert.assertEquals(numUpdates, lrListener.getUpdates().size());
         verifyData(lrListener.getUpdates(), lrListener.getExistingEntries());
-
-        // Verify that no data was observed on the listener corresponding to new_client
-        Assert.assertTrue(newListener.getUpdates().isEmpty());
```

Comment:
these asserts are valid, right?

Reply:
Yes, good catch I forgot to revert those after some debugging.